modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -130,25 +146,20 @@ Only expressions using dots as field separators are allowed. While `\#[payload.p
 | No. +
 Keep in mind that if you don't specify a selector expression, the policy will be applied, but won't tokenize anything from your payload.
 
-| *Format*
+| Format
 | Type of information that will be tokenized. Formats may vary depending on the selected service. Choose one format from the list.
 
-If you do not see any formats in the list, check that the tokenization service selected contains at least one format. +
-See the xref:anypoint-security::tokenization-formats.adoc[Tokenization Formats] article for more information.
+If you do not see any formats in the list, check that the tokenization service that you selected contains at least one
+pppp. 
```

Comment:
Will your audience know what "pppp" is?

Reply:
Seems like the format ought to be described further in the conceptual text.

Reply:
Done! Thanks!